### ISSUE TITLE
Grid rendering breaks when encountering invalid molblocks 

### ISSUE THREAD

**Issue by skalkin** [datagrok-ai/public#653]
Here, only the molblock in row 3 is invalid, but instead all following ones are rendered as invalid as well.

![image](https://user-images.githubusercontent.com/943403/170374974-295f6352-417b-475e-aa24-88c76e6603e1.png)

**Comment by skalkin** [comment]
GridCore

**Comment by skalkin** [comment]
Here's the structure that breaks the rendering:

```

  ACCLDraw05192210572D

  0  0  0     0  0            999 V3000
M  V30 BEGIN CTAB
M  V30 COUNTS 19 19 0 0 0
M  V30 BEGIN ATOM
M  V30 1 C 6.8837 -6.5521 0 0 
M  V30 2 C 5.55 -7.3221 0 0 
M  V30 3 C 5.55 -8.8621 0 0 
M  V30 4 C 6.8837 -9.6321 0 0 
M  V30 5 C 8.2173 -8.8621 0 0 
M  V30 6 C 8.1235 -7.1659 0 0 
M  V30 7 C 6.8837 -5.0121 0 0 CFG=2 
M  V30 8 C 8.2173 -4.2421 0 0 
M  V30 9 C 5.55 -4.2421 0 0 
M  V30 10 C 5.55 -2.7021 0 0 
M  V30 11 C 9.4632 -9.7673 0 0 CFG=2 
M  V30 12 C 10.8701 -9.1409 0 0 
M  V30 13 C 9.3022 -11.2989 0 0 
M  V30 14 C 10.5481 -12.2041 0 0 
M  V30 15 C 6.8837 -11.1721 0 0 
M  V30 16 C 9.4572 -6.3959 0 0 CFG=2 
M  V30 17 C 9.4572 -4.8559 0 0 
M  V30 18 C 10.7909 -7.1659 0 0 
M  V30 19 C 12.1245 -6.3959 0 0 
M  V30 END ATOM
M  V30 BEGIN BOND
M  V30 1 1 1 2 
M  V30 2 2 2 3 
M  V30 3 1 3 4 
M  V30 4 2 4 5 
M  V30 5 1 5 6 
M  V30 6 2 1 6 
M  V30 7 1 7 1 
M  V30 8 1 7 8 CFG=1 
M  V30 9 1 7 9 
M  V30 10 1 9 10 
M  V30 11 1 11 5 
M  V30 12 1 11 12 CFG=3 
M  V30 13 1 11 13 
M  V30 14 1 13 14 
M  V30 15 1 4 15 
M  V30 16 1 6 16 
M  V30 17 1 16 17 CFG=3 
M  V30 18 1 16 18 
M  V30 19 1 18 19 
M  V30 END BOND
M  V30 BEGIN COLLECTION
M  V30 MDLV30/STEABS ATOMS=(1 16)
M  V30 MDLV30/STERAC1 ATOMS=(2 7 11)
M  V30 END COLLECTION
M  V30 END CTAB
M  END
```

**Comment by dionissqq** [comment]
fixed in v1.4.13

**Comment by dionissqq** [comment]
fixed
